homeassistant/components/sungrow/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "sungrow",
+  "name": "Sungrow",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/sungrow",
+  "requirements": ["sungrow-modbus-client==0.0.2", "pymodbus==2.5.3"],
```

Comment:
I saw several other sungrow modus TCP clients, that seemed to have been around for a while already.

Is there a reason you chose to roll your own in this case?

Reply:
https://pypi.org/project/SungrowModbusTcpClient/
https://github.com/meltaxa/solariot

Reply:
I also found those two projects. But they didn't fit the needs I was having.
The SungrowModbusTcpClient does not provide a way to interpret the actual values and solariot is not a library but rather a service to load the information based on a config file.

Reply:
Ok 😃